compiler/crates/relay-codegen/src/printer.rs

```diff
@@ -661,7 +661,7 @@ fn get_module_path(js_module_format: JsModuleFormat, key: StringKey) -> Cow<'sta
             let extension = path.extension();
 
             if let Some(extension) = extension {
-                if extension == "ts" || extension == "js" {
+                if extension == "ts" || extension == "tsx" || extension === "js" {
```

Comment:
Shouldn't it be `extension == "js"` instead?

```suggestion
                if extension == "ts" || extension == "tsx" || extension == "js" {
```